test/demos.js

```diff
@@ -0,0 +1,112 @@
+const assert = require('chai').assert;
```

Comment:
Before we merge this (or before we merge chrome-headless, if that's where we did the deletes), we should squash or rewrite so we aren't removing most of the test files and then adding them back—it'll mess up our ability to use `git blame` and obscure the few changes we did make to those files.